src/coreclr/vm/comutilnative.cpp

```diff
@@ -1846,6 +1846,39 @@ extern "C" BOOL QCALLTYPE MethodTable_AreTypesEquivalent(MethodTable* mta, Metho
     return bResult;
 }
 
+extern "C" BOOL QCALLTYPE TypeHandle_CanCastTo(void* fromTypeHnd, void* toTypeHnd)
```

Comment:
```suggestion
extern "C" BOOL QCALLTYPE TypeHandle_CanCastToNoCacheLookup(void* fromTypeHnd, void* toTypeHnd)
```
I think this would be a better name for the QCall to make it clear what it does. Matches convention used for cast helpers (`ChkCastAny_NoCacheLookup`, etc.)